packages/block-library/src/navigation/index.php

```diff
@@ -277,6 +277,7 @@ function block_core_navigation_get_classic_menu_fallback() {
 		}
 
 		// Otherwise return the most recently created classic menu.
+		usort( $classic_nav_menus, fn( $a, $b) => $b->term_id - $a->term_id );
```

Comment:
I think this syntax was introduced in PHP 7.4. WordPress says it requires PHP 7.4, but I don't see any use of arrow functions in WordPress or Gutenberg, so maybe it's better to avoid them?

Reply:
Fixed in 3871774f423eed36338a57633418c3e8be4a3f6f